airbyte-connector-builder-server/pyproject.toml

```diff
@@ -0,0 +1,83 @@
+[tool.black]
```

Comment:
this file is a copy of https://github.com/airbytehq/airbyte/blob/master/pyproject.toml without the [skip_gitignore to remove the dependency on git](https://github.com/airbytehq/airbyte/blob/master/pyproject.toml#L44)

Reply:
Will it be annoying to maintain these two different files separately since they have so much duplication? I'm guessing there isn't an easy way to configure this to just ignore that one line?

Also totally fine if the answer is just "this is the quickest way to get V0 working"

Reply:
fair point. The reason I copied it is that I removed the dependency on git. Thinking more about it, I think it'll be easier to maintain if we just install git in the docker container and symlink to the original